data/transactions/logic/doc.go

```diff
@@ -257,21 +257,22 @@ var opDescByName = map[string]OpDesc{
-	"b-":  {"A minus B. A and B are interpreted as big-endian unsigned integers. Fail on underflow.", "", nil},
-	"b/":  {"A divided by B (truncated division). A and B are interpreted as big-endian unsigned integers. Fail if B is zero.", "", nil},
-	"b*":  {"A times B. A and B are interpreted as big-endian unsigned integers.", "", nil},
-	"b<":  {"1 if A is less than B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
-	"b>":  {"1 if A is greater than B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
-	"b<=": {"1 if A is less than or equal to B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
-	"b>=": {"1 if A is greater than or equal to B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
-	"b==": {"1 if A is equal to B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
-	"b!=": {"0 if A is equal to B, else 1. A and B are interpreted as big-endian unsigned integers", "", nil},
-	"b%":  {"A modulo B. A and B are interpreted as big-endian unsigned integers. Fail if B is zero.", "", nil},
-	"b|":  {"A bitwise-or B. A and B are zero-left extended to the greater of their lengths", "", nil},
-	"b&":  {"A bitwise-and B. A and B are zero-left extended to the greater of their lengths", "", nil},
-	"b^":  {"A bitwise-xor B. A and B are zero-left extended to the greater of their lengths", "", nil},
-	"b~":  {"A with all bits inverted", "", nil},
+	"b+":      {"A plus B. A and B are interpreted as big-endian unsigned integers", "", nil},
+	"b-":      {"A minus B. A and B are interpreted as big-endian unsigned integers. Fail on underflow.", "", nil},
+	"b/":      {"A divided by B (truncated division). A and B are interpreted as big-endian unsigned integers. Fail if B is zero.", "", nil},
+	"b*":      {"A times B. A and B are interpreted as big-endian unsigned integers.", "", nil},
+	"b<":      {"1 if A is less than B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
+	"b>":      {"1 if A is greater than B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
+	"b<=":     {"1 if A is less than or equal to B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
+	"b>=":     {"1 if A is greater than or equal to B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
+	"b==":     {"1 if A is equal to B, else 0. A and B are interpreted as big-endian unsigned integers", "", nil},
+	"b!=":     {"0 if A is equal to B, else 1. A and B are interpreted as big-endian unsigned integers", "", nil},
+	"b%":      {"A modulo B. A and B are interpreted as big-endian unsigned integers. Fail if B is zero.", "", nil},
+	"bmodexp": {"A raised to the Bth power modulo C. A, B and C are interpreted as big-endian unsigned integers limited to 4096 bytes. Fail if C is zero.", "", nil},
```

Comment:
Can you move this done to follow `bsqrt` both for consistency and because it will minimize the diff caused by autoformatting.